android/sdl_android/src/main/java/com/smartdevicelink/transport/utl/SdlDeviceListener.java

```diff
@@ -261,6 +268,44 @@ public static boolean isFeatureSupported(List<SdlAppInfo> sdlAppInfoList) {
         return true;
     }
 
+    public static void saveVehicleType(Context context, VehicleType vehicleType, String address) {
+        synchronized (LOCK) {
+
+            if (vehicleType == null || address == null) {
+                return;
+            }
+            try {
+                SharedPreferences preferences = context.getSharedPreferences(SDL_DEVICE_VEHICLES_PREFS, Context.MODE_PRIVATE);
+
+                String jsonString = vehicleType.serializeJSON().toString();
+                SharedPreferences.Editor editor = preferences.edit();
+                editor.putString(address, jsonString);
+                editor.commit();
+            } catch (JSONException e) {
+                Log.e(TAG, e.getMessage());
+            }
+        }
+    }
+
+    public static Hashtable<String, Object> getVehicleTypeFromPrefs(Context context, String address) {
```

Comment:
@kostyaBoss This method `getVehicleTypeFromPrefs()` on line 290 is not being used anywhere.  Do you need it or can it be deleted?

Reply:
@santhanamk just mirroring the second PR, let's keep it